btn/README.md

```diff
@@ -1,17 +1,34 @@
 ## Introduction
-This program prints a long string having no white-space characters, given a text file from stdin. Thus a would-be attention seeker may find this program useful for his doing so-called 'buntang.'
 
-## Build
-```
-make
-```
+This program prints a long string having no white-space characters, given a text file from stdin. Thus a would-be attention seeker may find this program useful for his doing so-called 'buntang,' hence the program name 'btn.'
 
 ## Usage
+
 ```
 ./btn N < TEXT_FILE_NAME
 
 where
    N = the number of characters before a new-line character;
        its range: 1 ~ 32,
    TEXT_FILE_NAME = the file name to be input-redirected.
+```
+
+## Implementations
+
+### C
+
+```
+make
+```
+
+### Python
+
+```
+python btn.py
```

Comment:
This line can lead users to misunderstand that there is no argument for the python program.
Suggestions:
- State that python does not compile.
- Make clear that there's one argument.
```
python btn.py N
```

Reply:
![image](https://github.com/sdbx/Utils/assets/83723320/8d0abd2f-751f-4e38-aeff-21c290345e89)

A nice detection!

Reply:
The similar problem is in nsy/README.md, so I'm going to fix that too with it.

Reply:
Incidentally, I think it's worth noticing that the current btn/nsy implementations in python does not take its input from stdin.